pkg/controlplane/controller/kubernetesservice/controller.go

```diff
@@ -267,7 +233,12 @@ func (c *Controller) UpdateKubernetesService(reconcile bool) error {
 	// TODO: when it becomes possible to change this stuff,
 	// stop polling and start watching.
 	// TODO: add endpoints of all replicas, not just the elected master.
-	if err := createNamespaceIfNeeded(c.client.CoreV1(), metav1.NamespaceDefault); err != nil {
+	if _, err := c.Client.CoreV1().Namespaces().Create(context.TODO(), &corev1.Namespace{
```

Comment:
we move from "Get and Create" if not exist to "Create and continue if exist or is created succesfully", is there any implication on doing this periodically?

Reply:
I think I prefer the former, because this will show as apiserver errors in the logs, I think that is less noisier

Reply:
The behaviour from the client point of view is the same. But this create will be an additional write, while the get is served from cache. Let me undo that change.

Reply:
done